docs/configuring-metabase/caching.md

```diff
@@ -7,13 +7,27 @@ redirect_from:
 
 # Caching query results
 
-If your question results don't change frequently, you may want to store the results so that the next time anyone visits the question, Metabase can retrieve the stored results rather than query the database again.
+If your question results don't change frequently, you may want to store the results so that the next time anyone visits the question, Metabase can retrieve the cached results rather than query the database again.
 
 For example, if your data only updates once a day, there's no point in querying the database more than once a day, as the data won't have changed. Returning cached results can be significantly faster, as the database won't have to recompute the results to load your question.
 
+You can set [caching invalidation policies](#cache-invalidation-policies) for [questions](#question-caching-policy), [dashboards](#dashboard-caching-policy), and [databases](#database-caching-policy).
+
+## How caching works in Metabase
+
+Let's say you set a caching policy for a particular question. You set a [duration](#duration-caching-policy) that says to invalidate the results after one hour.
+
+When you view the question for the first time, Metabase will check for stored results. When it doesn't find any, it will run the question, return the results, and store (cache) the results. Those results will remain valid for the next hour (according to the hour-long duration policy you set).
+
+You run the question half an hour later, Metabase will return those stored results.
```

Comment:
```suggestion
If you run the question half an hour later, Metabase will return those stored results.
```

I realize what you have is good, but I figure this is a lil bit harder to misunderstand